### DISCUSSION TITLE
How to Mark Reconciled Transactions as Unreconciled

### DISCUSSION THREAD

**Discussion by softpoint-arch** [firefly-iii/firefly-iii#9576] [Firefly III]
If I would like to undo my previous reconciliation, how do I mark reconciled transactions as unreconciled?

**Comment by JC5** [comment]
Sorry, at the moment you cannot do this.

**Reply by bfly75** [reply]
Would it be possible to do this by editing the database directly (by setting the reconciled bit of the transaction in the transactions table to FALSE)? Or does that have all kind of unforeseen consequences?

**Reply by JC5** [reply]
No that would work, both in `transactions` and `transaction_journals`.
